.travis.yml

```diff
@@ -26,6 +26,9 @@ env:
   - BUILD=site
   - BUILD=sonar
 
+matrix:
+  fast_finish: true
```

Comment:
It probably won't hurt, if we set this property. The problems discussed in the travis issue shouldn't apply to use, as we run a single script per step (.build-$BUILD.sh) which uses the "set -e" bash option, to exit with the first failed command. However, the 3 scripts are started in parallel...
Deployment is in the deployment section and only is executed, if all matrix-builds succeeded.